experiments/calibration/shared/make_training_locations.jl

```diff
@@ -0,0 +1,19 @@
+# Define the locations longitudes and latitudes
+# Needs to be defined once, both for g(θ) and ERA5 target
+t0 = 0.0
+tf = 60 * 60.0 * 24 * 366
+Δt = 900.0
+nelements = (50, 5)
```

Comment:
I'm concerned about the redefinitions of these objects. Especially given that they are supposed to be in a "Shared" file, they are redefined within the experiments.